app/jobs/cancel_reservations_job.rb

```diff
@@ -3,10 +3,12 @@ class CancelReservationsJob < ApplicationJob
 
   def perform
     #find all screenings that will start in 30 minuts or earlier
-    screenings  = Screenings::UseCases::FindAll.new.call
+    #maybe it can be done in one call? for reasearch
+    screenings  = Screenings::Repository.new.today
```

Comment:
Maybe create a repository method that returns screenings that have date in 30 from now.  Something like this may work: 
```
adapter.where(date: Time.now - 30.minutes..Time.now)
```

Reply:
Działa ale w wersji `adapter.where(date: Time.now..Time.now + 30.minutes)` ;)